install_config/install/prerequisites.adoc

```diff
@@ -805,3 +805,61 @@ Following the installation process, you can configure {product-title} for
 xref:../../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS],
 xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[OpenStack], or
 xref:../../install_config/configuring_gce.adoc#install-config-configuring-gce[GCE].
+
+[[prereq-containerized-glusterfs-considerations]]
+=== Containerized GlusterFS Considerations
+
+If you choose to configure
+xref:../../install_config/install/advanced_install.adoc#advanced-install-containerized-glusterfs-persistent-storage[containerized GlusterFS persistent storage] for your cluster, or if you choose to
+configure a
+xref:../../install_config/install/advanced_install.adoc#advanced-install-glusterfs-backed-registry[containerized GlusterFS-backed OpenShift Container Registry], you must consider the following prerequisites.
+
+[[prereq-glusterfs-storage-nodes]]
+==== Storage Nodes
+
+To use containerized GlusterFS persistent storage:
+
+- Each storage node is recommended to have at least 1 raw block device with least
+100 GB available.
+
+To run a containerized GlusterFS-backed OpenShift Container Registry:
+
+- A minimum of 3 storage nodes must be available for 3-way replication.
+- Each storage node must have at least 1 raw block device with at least 10 GB of
+free storage. 
+
+[IMPORTANT]
+====
+While containerized GlusterFS persistent storage can be configured and deployed
+on the same {product-title} cluster as a containerized GlusterFS-backed
+registry, their storage should be kept separate from each other and also
+requires additional storage nodes. For example, if both are configured, a total
+of 6 storage nodes would be needed: 3 for the registry and 3 for persistent
```

Comment:
Here I have less of a leg to stand on since it's more clearly its own paragraph.